lib/cloud_controller/diego/protocol.rb

```diff
@@ -66,7 +66,7 @@ def desire_app_message(process, default_health_check_timeout)
           'etag'                            => process.updated_at.to_f.to_s,
           'allow_ssh'                       => process.enable_ssh,
           'ports'                           => OpenProcessPorts.new(process).to_a,
-          'network'                         => ContainerNetworkInfo.new(process.app).to_h,
+          'network'                         => ContainerNetworkInfo.new(process.app, 'desired app message').to_h,
```

Comment:
this looks wrong to me, should this be `Protocol::ContainerNetworkInfo::APP`?

Reply:
you got it amigo